src/common/darktable.c

```diff
@@ -129,9 +129,10 @@ static int usage(const char *argv0)
   printf("  --cachedir <user cache directory>\n");
   printf("  --conf <key>=<value>\n");
   printf("  --configdir <user config directory>\n");
-  printf("  -d {all,act_on,cache,camctl,camsupport,control,dev,imageio,\n");
+  printf("  -d {act_on,cache,camctl,camsupport,control,dev,imageio,\n");
   printf("      input,ioporder,lighttable,lua,masks,memory,nan,opencl,params,\n");
-  printf("      perf,print,pwstorage,signal,sql,tiling,undo,verbose,pipe}\n");
+  printf("      perf,print,pwstorage,signal,sql,tiling,undo,verbose,pipe,\n");
+  printf("      all,common}\n");
   printf("  --d-signal <signal> \n");
```

Comment:
Maybe worth noting that common = opencl + dev + masks + params + imageio + pipe ?

Reply:
right